dataflow/java/src/main/java/com/httparchive/dataflow/BigQueryImport.java

```diff
@@ -288,20 +318,19 @@ public static void main(String[] args) {
                 DataflowPipelineWorkerPoolOptions.AutoscalingAlgorithmType.NONE);
 
         Pipeline p = Pipeline.create(pipelineOptions);
-
-        PCollectionTuple results = p.apply(TextIO.Read
-                .named("read-har")
-                .from(getHarBucket(options))
-                .withCompressionType(TextIO.CompressionType.GZIP)
-                .withCoder(HarJsonCoder.of()))
+        PCollectionTuple results = p
```

Comment:
The logic that you have in `HarJsonCoder` is the same logic you'll want here. Just read and decompress the file and parse it to `JsonNode` using the same sort of `ObjectMapper` that is set up there.